charts/rancher-monitoring/v0.2.1/questions.yml

```diff
@@ -1 +1 @@
-rancher_min_version: 2.5.4-rc1
+rancher_min_version: 2.4.13-rc1
```

Comment:
Required since `2.4.13` is the first Rancher release in the `2.4` line that respects `kubeVersion` on a Helm chart on deploy.